src/Conda.jl

```diff
@@ -34,43 +34,95 @@ module Conda
 using Compat
 import Compat.String
 using JSON
+include("../deps/deps.jl")
 
-"Prefix for installation of all the packages."
-const PREFIX = abspath(dirname(@__FILE__), "..", "deps", "usr")
+if !isdir(default_dir)
+    # Ensure default_dir exists
+    mkpath(default_dir)
+end
+
+immutable Environment
```

Comment:
You could just do `typealias Environment Union{AbstractString,Symbol}`.   The only place you would actually need to convert the `Symbol` to a path would be in a `_set_conda_env(cmd, env::Symbol)` method.